src/Moonglade.Auditing/MoongladeAudit.cs

```diff
@@ -96,21 +95,32 @@ public async Task<Response> AddAuditEntry(EventType eventType, EventId eventId,
                             AND (@EventId IS NULL OR al.EventId = @EventId)
                             ORDER BY al.EventTimeUtc DESC
                             OFFSET @Skip ROWS
-                            FETCH NEXT @Take ROWS ONLY";
+                            FETCH NEXT @Take ROWS ONLY
 
-                var entries = await conn.QueryAsync<AuditEntry>(sql, new
+                            SELECT COUNT(al.Id)
```

Comment:
Seems you are using the second SQL to get total counts.

Reply:
This is right, it's QueryMutilple feature to enhance performance